### ISSUE TITLE
Assets empty after v5 upgrade

### ISSUE THREAD

**Issue by cliff-LINKS** [grokability/snipe-it#8604]
[laravel.log](https://github.com/snipe/snipe-it/files/5431826/laravel.log)
#### Please confirm you have done the following before posting your bug report:

- [V] I have enabled debug mode 
- [V] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

After an upgrade from 4.9.5 to 5.0.1 my assets list shows up empty. All other items are still OK.
The records are still there in my SQL db.
I can still see and edit the assets records via the View All activity list in the Dashboard.

Later today v5.0.2 was released, i've run the update to no avail.

**Server (please complete the following information):**
 - Snipe-IT Version 5.0.2
 - OS: macOS 10.13.6
 - Web Server: Apache
 - PHP Version: 7.1.33

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Safari
 - Version: 14.0

**Error Messages**
- None

**Additional context**
- This is an upgrade from 4.9.5 to 5.0.1 from an existing installation
- The upgrade was done via php upgrade.php
- As I found in previous mentions of a similar issue, I've removed all ' and " occurrences in fields of the Assets table, to no avail.
- The database was used to check the contents, no data was manually altered.

It looks to me that all entries in the laravel.log are created before the installation of the upgrade.

**Comment by snipe** [comment]
Include any additional information you can find in `storage/logs`, your browser console, and your webserver's logs.

**Comment by cliff-LINKS** [comment]
I get this gibberish in the console of my browser if I hit the Assets button.

https://mail.linksit.be:44344/api/v1/hardware?status=&order_number=&company_id=&status_id=&search=&sort=name&order=desc&offset=0&limit=50&searchable%5B%5D=company&searchable%5B%5D=name&searchable%5B%5D=asset_tag&searchable%5B%5D=serial&searchable%5B%5D=model&searchable%5B%5D=model_number&searchable%5B%5D=category&searchable%5B%5D=status_label&searchable%5B%5D=assigned_to&searchable%5B%5D=location&searchable%5B%5D=rtd_location&searchable%5B%5D=manufacturer&searchable%5B%5D=supplier&searchable%5B%5D=purchase_date&searchable%5B%5D=purchase_cost&searchable%5B%5D=order_number&searchable%5B%5D=warranty_months&searchable%5B%5D=notes&searchable%5B%5D=custom_fields._snipeit_port_01_mac_address_1&searchable%5B%5D=custom_fields._snipeit_memory_type_4&searchable%5B%5D=custom_fields._snipeit_cpu_3&searchable%5B%5D=custom_fields._snipeit_memory_slots_5&searchable%5B%5D=custom_fields._snipeit_memory_modules_6&searchable%5B%5D=custom_fields._snipeit_local_user_1_name_7&searchable%5B%5D=custom_fields._snipeit_local_user_1_password_8&searchable%5B%5D=custom_fields._snipeit_local_user_2_name_9&searchable%5B%5D=custom_fields._snipeit_local_user_2_password_10&searchable%5B%5D=custom_fields._snipeit_port_01_type_12&searchable%5B%5D=custom_fields._snipeit_port_01_ip_11&searchable%5B%5D=custom_fields._snipeit_port_01_dns_13&searchable%5B%5D=custom_fields._snipeit_port_01_dhcp_15&searchable%5B%5D=custom_fields._snipeit_port_01_subnet_14&searchable%5B%5D=custom_fields._snipeit_gpu_16&searchable%5B%5D=custom_fields._snipeit_storage_17&searchable%5B%5D=custom_fields._snipeit_os_18&searchable%5B%5D=custom_fields._snipeit_port_01_vlan_id_19&searchable%5B%5D=custom_fields._snipeit_port_01_speed_20&searchable%5B%5D=custom_fields._snipeit_port_01_router_21&searchable%5B%5D=custom_fields._snipeit_port_02_type_22&searchable%5B%5D=custom_fields._snipeit_port_02_dhcp_23&searchable%5B%5D=custom_fields._snipeit_port_02_ip_24&searchable%5B%5D=custom_fields._snipeit_port_02_subnet_25&searchable%5B%5D=custom_fields._snipeit_port_02_router_26&searchable%5B%5D=custom_fields._snipeit_port_02_dns_27&searchable%5B%5D=custom_fields._snipeit_port_02_speed_28&searchable%5B%5D=custom_fields._snipeit_port_02_mac_address_30&searchable%5B%5D=custom_fields._snipeit_port_02_vlan_id_29&searchable%5B%5D=custom_fields._snipeit_port_03_type_31&searchable%5B%5D=custom_fields._snipeit_port_03_dhcp_32&searchable%5B%5D=custom_fields._snipeit_port_03_ip_33&searchable%5B%5D=custom_fields._snipeit_port_03_subnet_34&searchable%5B%5D=custom_fields._snipeit_port_03_router_35&searchable%5B%5D=custom_fields._snipeit_port_03_speed_37&searchable%5B%5D=custom_fields._snipeit_port_03_dns_36&searchable%5B%5D=custom_fields._snipeit_port_03_vlan_id_38&searchable%5B%5D=custom_fields._snipeit_port_03_mac_address_39&searchable%5B%5D=custom_fields._snipeit_port_04_type_40&searchable%5B%5D=custom_fields._snipeit_port_04_dhcp_41&searchable%5B%5D=custom_fields._snipeit_port_04_ip_42&searchable%5B%5D=custom_fields._snipeit_port_04_subnet_43&searchable%5B%5D=custom_fields._snipeit_port_04_router_44&searchable%5B%5D=custom_fields._snipeit_port_04_dns_45&searchable%5B%5D=custom_fields._snipeit_port_04_speed_46&searchable%5B%5D=custom_fields._snipeit_port_04_vlan_id_47&searchable%5B%5D=custom_fields._snipeit_port_04_mac_address_48&searchable%5B%5D=custom_fields._snipeit_port_05_type_49&searchable%5B%5D=custom_fields._snipeit_port_05_dhcp_50&searchable%5B%5D=custom_fields._snipeit_port_05_ip_51&searchable%5B%5D=custom_fields._snipeit_port_05_subnet_52&searchable%5B%5D=custom_fields._snipeit_port_05_router_53&searchable%5B%5D=custom_fields._snipeit_port_05_dns_54&searchable%5B%5D=custom_fields._snipeit_port_05_speed_55&searchable%5B%5D=custom_fields._snipeit_port_05_vlan_id_56&searchable%5B%5D=custom_fields._snipeit_port_05_mac_address_57&searchable%5B%5D=custom_fields._snipeit_port_06_type_58&searchable%5B%5D=custom_fields._snipeit_port_06_dhcp_59&searchable%5B%5D=custom_fields._snipeit_port_06_ip_60&searchable%5B%5D=custom_fields._snipeit_port_06_subnet_61&searchable%5B%5D=custom_fields._snipeit_port_06_router_62&searchable%5B%5D=custom_fields._snipeit_port_06_dns_63&searchable%5B%5D=custom_fields._snipeit_port_06_speed_64&searchable%5B%5D=custom_fields._snipeit_port_06_vlan_id_65&searchable%5B%5D=custom_fields._snipeit_port_06_mac_address_66&searchable%5B%5D=custom_fields._snipeit_port_07_type_67&searchable%5B%5D=custom_fields._snipeit_port_07_dhcp_68&searchable%5B%5D=custom_fields._snipeit_port_07_ip_69&searchable%5B%5D=custom_fields._snipeit_port_07_subnet_70&searchable%5B%5D=custom_fields._snipeit_port_07_router_71&searchable%5B%5D=custom_fields._snipeit_port_07_dns_72&searchable%5B%5D=custom_fields._snipeit_port_07_speed_73&searchable%5B%5D=custom_fields._snipeit_port_07_vlan_id_74&searchable%5B%5D=custom_fields._snipeit_port_07_mac_address_75&searchable%5B%5D=custom_fields._snipeit_port_08_type_76&searchable%5B%5D=custom_fields._snipeit_port_08_dhcp_77&searchable%5B%5D=custom_fields._snipeit_port_08_ip_78&searchable%5B%5D=custom_fields._snipeit_port_08_subnet_79&searchable%5B%5D=custom_fields._snipeit_port_08_router_80&searchable%5B%5D=custom_fields._snipeit_port_08_dns_81&searchable%5B%5D=custom_fields._snipeit_port_08_speed_82&searchable%5B%5D=custom_fields._snipeit_port_08_vlan_id_83&searchable%5B%5D=custom_fields._snipeit_port_08_mac_address_84&searchable%5B%5D=custom_fields._snipeit_port_09_type_85&searchable%5B%5D=custom_fields._snipeit_port_09_dhcp_86&searchable%5B%5D=custom_fields._snipeit_port_09_ip_87&searchable%5B%5D=custom_fields._snipeit_port_09_subnet_88&searchable%5B%5D=custom_fields._snipeit_port_09_router_89&searchable%5B%5D=custom_fields._snipeit_port_09_dns_90&searchable%5B%5D=custom_fields._snipeit_port_09_speed_91&searchable%5B%5D=custom_fields._snipeit_port_09_vlan_id_92&searchable%5B%5D=custom_fields._snipeit_port_09_mac_address_93&searchable%5B%5D=custom_fields._snipeit_port_10_type_94&searchable%5B%5D=custom_fields._snipeit_port_10_dhcp_95&searchable%5B%5D=custom_fields._snipeit_port_10_ip_96&searchable%5B%5D=custom_fields._snipeit_port_10_subnet_97&searchable%5B%5D=custom_fields._snipeit_port_10_router_98&searchable%5B%5D=custom_fields._snipeit_port_10_dns_99&searchable%5B%5D=custom_fields._snipeit_port_10_speed_100&searchable%5B%5D=custom_fields._snipeit_port_10_vlan_id_101&searchable%5B%5D=custom_fields._snipeit_port_10_mac_address_102&searchable%5B%5D=custom_fields._snipeit_port_11_type_103&searchable%5B%5D=custom_fields._snipeit_port_11_dhcp_104&searchable%5B%5D=custom_fields._snipeit_port_11_ip_105&searchable%5B%5D=custom_fields._snipeit_port_11_subnet_106&searchable%5B%5D=custom_fields._snipeit_port_11_router_107&searchable%5B%5D=custom_fields._snipeit_port_11_dns_108&searchable%5B%5D=custom_fields._snipeit_port_11_speed_109&searchable%5B%5D=custom_fields._snipeit_port_11_vlan_id_110&searchable%5B%5D=custom_fields._snipeit_port_11_mac_address_111&searchable%5B%5D=custom_fields._snipeit_port_12_type_112&searchable%5B%5D=custom_fields._snipeit_port_12_dhcp_113&searchable%5B%5D=custom_fields._snipeit_port_12_ip_114&searchable%5B%5D=custom_fields._snipeit_port_12_router_116&searchable%5B%5D=custom_fields._snipeit_port_12_subnet_115&searchable%5B%5D=custom_fields._snipeit_port_12_dns_117&searchable%5B%5D=custom_fields._snipeit_port_12_speed_118&searchable%5B%5D=custom_fields._snipeit_port_12_vlan_id_119&searchable%5B%5D=custom_fields._snipeit_port_12_mac_address_120&searchable%5B%5D=custom_fields._snipeit_port_01_hostname_121&searchable%5B%5D=custom_fields._snipeit_port_02_hostname_122&searchable%5B%5D=custom_fields._snipeit_port_03_hostname_123&searchable%5B%5D=custom_fields._snipeit_port_04_hostname_124&searchable%5B%5D=custom_fields._snipeit_port_05_hostname_125&searchable%5B%5D=custom_fields._snipeit_port_06_hostname_126&searchable%5B%5D=custom_fields._snipeit_port_01_dhcp_address_range_133&searchable%5B%5D=custom_fields._snipeit_port_02_dhcp_address_range_134&searchable%5B%5D=custom_fields._snipeit_port_03_dhcp_address_range_135&searchable%5B%5D=custom_fields._snipeit_port_04_dhcp_address_range_136&searchable%5B%5D=custom_fields._snipeit_port_05_dhcp_address_range_137&searchable%5B%5D=custom_fields._snipeit_port_06_dhcp_address_range_138&searchable%5B%5D=custom_fields._snipeit_port_07_hostname_127&searchable%5B%5D=custom_fields._snipeit_port_07_dhcp_address_range_139&searchable%5B%5D=custom_fields._snipeit_port_08_hostname_128&searchable%5B%5D=custom_fields._snipeit_port_08_dhcp_address_range_140&searchable%5B%5D=custom_fields._snipeit_port_09_hostname_129&searchable%5B%5D=custom_fields._snipeit_port_09_dhcp_address_range_141&searchable%5B%5D=custom_fields._snipeit_port_10_hostname_130&searchable%5B%5D=custom_fields._snipeit_port_10_dhcp_address_range_142&searchable%5B%5D=custom_fields._snipeit_port_11_hostname_131&searchable%5B%5D=custom_fields._snipeit_port_11_dhcp_address_range_143&searchable%5B%5D=custom_fields._snipeit_port_12_hostname_132&searchable%5B%5D=custom_fields._snipeit_port_12_dhcp_address_range_144&searchable%5B%5D=custom_fields._snipeit_management_url_1_145&searchable%5B%5D=custom_fields._snipeit_management_url_2_146&searchable%5B%5D=custom_fields._snipeit_management_url_3_147&searchable%5B%5D=custom_fields._snipeit_support_contract_type_148&searchable%5B%5D=custom_fields._snipeit_support_contract_end_date_149&searchable%5B%5D=custom_fields._snipeit_support_account_login_150&searchable%5B%5D=custom_fields._snipeit_support_account_password_151&searchable%5B%5D=custom_fields._snipeit_ssid_01_152&searchable%5B%5D=custom_fields._snipeit_ssid_01_password_153&searchable%5B%5D=custom_fields._snipeit_ssid_01_encryptie_154&searchable%5B%5D=custom_fields._snipeit_ssid_01_speed_155&searchable%5B%5D=custom_fields._snipeit_ssid_01_freq_156&searchable%5B%5D=custom_fields._snipeit_ssid_01_channel_157&searchable%5B%5D=custom_fields._snipeit_ssid_01_vlan_id_158&searchable%5B%5D=custom_fields._snipeit_ssid_02_159&searchable%5B%5D=custom_fields._snipeit_ssid_02_password_160&searchable%5B%5D=custom_fields._snipeit_ssid_02_encryptie_161&searchable%5B%5D=custom_fields._snipeit_ssid_02_speed_162&searchable%5B%5D=custom_fields._snipeit_ssid_02_freq_163&searchable%5B%5D=custom_fields._snipeit_ssid_02_channel_164&searchable%5B%5D=custom_fields._snipeit_ssid_02_vlan_id_165&searchable%5B%5D=custom_fields._snipeit_routing_166&searchable%5B%5D=custom_fields._snipeit_remote_support_id_167&searchable%5B%5D=custom_fields._snipeit_remote_support_password_168&searchable%5B%5D=custom_fields._snipeit_vpn_psck_169&searchable%5B%5D=custom_fields._snipeit_directory_domain_170&searchable%5B%5D=custom_fields._snipeit_netbios_realm_171&searchable%5B%5D=custom_fields._snipeit_directory_type_172&searchable%5B%5D=custom_fields._snipeit_ldap_search_base_173&searchable%5B%5D=custom_fields._snipeit_diradmin_short_name_174&searchable%5B%5D=custom_fields._snipeit_diradmin_full_name_175&searchable%5B%5D=custom_fields._snipeit_diradmin_uid_176&searchable%5B%5D=custom_fields._snipeit_diradmin_password_178&searchable%5B%5D=custom_fields._snipeit_diradmin_ldap_path_179&searchable%5B%5D=custom_fields._snipeit_diradmin_group_membership_180&searchable%5B%5D=custom_fields._snipeit_diradmin_guid_177&searchable%5B%5D=custom_fields._snipeit_dir_user_1_full_name_181&searchable%5B%5D=custom_fields._snipeit_dir_user_1_short_name_182&searchable%5B%5D=custom_fields._snipeit_dir_user_1_password_184&searchable%5B%5D=custom_fields._snipeit_dir_user_1_uid_185&searchable%5B%5D=custom_fields._snipeit_dir_user_1_guid_186&searchable%5B%5D=custom_fields._snipeit_dir_user_1_ldap_path_187&searchable%5B%5D=custom_fields._snipeit_dir_user_1_group_membership_188&searchable%5B%5D=custom_fields._snipeit_dir_user_2_full_name_189&searchable%5B%5D=custom_fields._snipeit_dir_user_2_short_name_190&searchable%5B%5D=custom_fields._snipeit_dir_user_2_password_191&searchable%5B%5D=custom_fields._snipeit_dir_user_2_uid_192&searchable%5B%5D=custom_fields._snipeit_dir_user_2_guid_193&searchable%5B%5D=custom_fields._snipeit_dir_user_2_ldap_path_194&searchable%5B%5D=custom_fields._snipeit_dir_user_2_group_membership_195&searchable%5B%5D=custom_fields._snipeit_dir_user_3_full_name_196&searchable%5B%5D=custom_fields._snipeit_dir_user_3_short_name_197&searchable%5B%5D=custom_fields._snipeit_dir_user_3_password_198&searchable%5B%5D=custom_fields._snipeit_dir_user_3_uid_199&searchable%5B%5D=custom_fields._snipeit_dir_user_3_guid_200&searchable%5B%5D=custom_fields._snipeit_dir_user_3_ldap_path_201&searchable%5B%5D=custom_fields._snipeit_dir_user_3_group_membership_202&searchable%5B%5D=custom_fields._snipeit_dir_user_4_full_name_203&searchable%5B%5D=custom_fields._snipeit_dir_user_4_short_name_204&searchable%5B%5D=custom_fields._snipeit_dir_user_4_password_205&searchable%5B%5D=custom_fields._snipeit_dir_user_4_uid_206&searchable%5B%5D=custom_fields._snipeit_dir_user_4_guid_207&searchable%5B%5D=custom_fields._snipeit_dir_user_4_ldap_path_208&searchable%5B%5D=custom_fields._snipeit_dir_user_4_group_membership_209&searchable%5B%5D=custom_fields._snipeit_dir_user_5_full_name_210&searchable%5B%5D=custom_fields._snipeit_dir_user_5_short_name_211&searchable%5B%5D=custom_fields._snipeit_dir_user_5_password_213&searchable%5B%5D=custom_fields._snipeit_dir_user_5_uid_214&searchable%5B%5D=custom_fields._snipeit_dir_user_5_guid_215&searchable%5B%5D=custom_fields._snipeit_dir_user_5_ldap_path_216&searchable%5B%5D=custom_fields._snipeit_dir_user_5_group_membership_217&searchable%5B%5D=custom_fields._snipeit_local_user_3_password_219&searchable%5B%5D=custom_fields._snipeit_local_user_3_name_218&searchable%5B%5D=custom_fields._snipeit_ssid_03_220&searchable%5B%5D=custom_fields._snipeit_ssid_03_password_221&searchable%5B%5D=custom_fields._snipeit_ssid_03_encryptie_222&searchable%5B%5D=custom_fields._snipeit_ssid_03_speed_223&searchable%5B%5D=custom_fields._snipeit_ssid_03_freq_224&searchable%5B%5D=custom_fields._snipeit_ssid_03_channel_225&searchable%5B%5D=custom_fields._snipeit_ssid_03_vlan_id_226&searchable%5B%5D=custom_fields._snipeit_ssid_05_227&searchable%5B%5D=custom_fields._snipeit_ssid_05_password_228&searchable%5B%5D=custom_fields._snipeit_ssid_05_encryptie_229&searchable%5B%5D=custom_fields._snipeit_ssid_05_speed_230&searchable%5B%5D=custom_fields._snipeit_ssid_05_freq_231&searchable%5B%5D=custom_fields._snipeit_ssid_05_channel_232&searchable%5B%5D=custom_fields._snipeit_ssid_05_vlan_id_233&searchable%5B%5D=custom_fields._snipeit_ssid_04_241&searchable%5B%5D=custom_fields._snipeit_ssid_04_password_242&searchable%5B%5D=custom_fields._snipeit_ssid_04_encryptie_243&searchable%5B%5D=custom_fields._snipeit_ssid_04_speed_244&searchable%5B%5D=custom_fields._snipeit_ssid_04_freq_245&searchable%5B%5D=custom_fields._snipeit_ssid_04_channel_246&searchable%5B%5D=custom_fields._snipeit_ssid_04_vlan_id_247&searchable%5B%5D=custom_fields._snipeit_ssid_06_234&searchable%5B%5D=custom_fields._snipeit_ssid_06_password_235&searchable%5B%5D=custom_fields._snipeit_ssid_06_encryptie_236&searchable%5B%5D=custom_fields._snipeit_ssid_06_speed_237&searchable%5B%5D=custom_fields._snipeit_ssid_06_freq_238&searchable%5B%5D=custom_fields._snipeit_ssid_06_channel_239&searchable%5B%5D=custom_fields._snipeit_ssid_06_vlan_id_240&searchable%5B%5D=custom_fields._snipeit_dir_user_1_email_248&searchable%5B%5D=custom_fields._snipeit_dir_user_2_email_249&searchable%5B%5D=custom_fields._snipeit_dir_user_3_email_250&searchable%5B%5D=custom_fields._snipeit_dir_user_4_email_251&searchable%5B%5D=custom_fields._snipeit_dir_user_5_email_252&searchable%5B%5D=custom_fields._snipeit_dir_user_6_full_name_253&searchable%5B%5D=custom_fields._snipeit_dir_user_6_short_name_254&searchable%5B%5D=custom_fields._snipeit_dir_user_6_password_255&searchable%5B%5D=custom_fields._snipeit_dir_user_6_email_258&searchable%5B%5D=custom_fields._snipeit_dir_user_6_uid_256&searchable%5B%5D=custom_fields._snipeit_dir_user_6_guid_257&searchable%5B%5D=custom_fields._snipeit_dir_user_6_ldap_path_259&searchable%5B%5D=custom_fields._snipeit_dir_user_6_group_membership_260&searchable%5B%5D=custom_fields._snipeit_dir_user_7_full_name_261&searchable%5B%5D=custom_fields._snipeit_dir_user_7_short_name_262&searchable%5B%5D=custom_fields._snipeit_dir_user_7_password_263&searchable%5B%5D=custom_fields._snipeit_dir_user_7_email_266&searchable%5B%5D=custom_fields._snipeit_dir_user_7_uid_264&searchable%5B%5D=custom_fields._snipeit_dir_user_7_guid_265&searchable%5B%5D=custom_fields._snipeit_dir_user_7_ldap_path_267&searchable%5B%5D=custom_fields._snipeit_dir_user_7_group_membership_268&searchable%5B%5D=custom_fields._snipeit_dir_user_8_full_name_269&searchable%5B%5D=custom_fields._snipeit_dir_user_8_short_name_270&searchable%5B%5D=custom_fields._snipeit_dir_user_8_password_271&searchable%5B%5D=custom_fields._snipeit_dir_user_8_email_272&searchable%5B%5D=custom_fields._snipeit_dir_user_8_uid_273&searchable%5B%5D=custom_fields._snipeit_dir_user_8_guid_274&searchable%5B%5D=custom_fields._snipeit_dir_user_8_ldap_path_275&searchable%5B%5D=custom_fields._snipeit_dir_user_8_group_membership_276&searchable%5B%5D=custom_fields._snipeit_dir_user_9_full_name_277&searchable%5B%5D=custom_fields._snipeit_dir_user_9_short_name_278&searchable%5B%5D=custom_fields._snipeit_dir_user_9_password_279&searchable%5B%5D=custom_fields._snipeit_dir_user_9_email_280&searchable%5B%5D=custom_fields._snipeit_dir_user_9_uid_281&searchable%5B%5D=custom_fields._snipeit_dir_user_9_guid_282&searchable%5B%5D=custom_fields._snipeit_dir_user_9_ldap_path_283&searchable%5B%5D=custom_fields._snipeit_dir_user_9_group_membership_284&searchable%5B%5D=custom_fields._snipeit_dir_user_10_full_name_285&searchable%5B%5D=custom_fields._snipeit_dir_user_10_short_name_286&searchable%5B%5D=custom_fields._snipeit_dir_user_10_password_287&searchable%5B%5D=custom_fields._snipeit_dir_user_10_email_288&searchable%5B%5D=custom_fields._snipeit_dir_user_10_uid_289&searchable%5B%5D=custom_fields._snipeit_dir_user_10_guid_290&searchable%5B%5D=custom_fields._snipeit_dir_user_10_ldap_path_291&searchable%5B%5D=custom_fields._snipeit_dir_user_10_group_membership_292&searchable%5B%5D=custom_fields._snipeit_dir_user_11_full_name_293&searchable%5B%5D=custom_fields._snipeit_dir_user_11_short_name_294&searchable%5B%5D=custom_fields._snipeit_dir_user_11_password_295&searchable%5B%5D=custom_fields._snipeit_dir_user_11_email_296&searchable%5B%5D=custom_fields._snipeit_dir_user_11_uid_297&searchable%5B%5D=custom_fields._snipeit_dir_user_11_guid_298&searchable%5B%5D=custom_fields._snipeit_dir_user_11_ldap_path_299&searchable%5B%5D=custom_fields._snipeit_dir_user_11_group_membership_300&searchable%5B%5D=custom_fields._snipeit_dir_user_12_full_name_301&searchable%5B%5D=custom_fields._snipeit_dir_user_12_short_name_302&searchable%5B%5D=custom_fields._snipeit_dir_user_12_password_303&searchable%5B%5D=custom_fields._snipeit_dir_user_12_email_304&searchable%5B%5D=custom_fields._snipeit_dir_user_12_uid_305&searchable%5B%5D=custom_fields._snipeit_dir_user_12_guid_306&searchable%5B%5D=custom_fields._snipeit_dir_user_12_ldap_path_307&searchable%5B%5D=custom_fields._snipeit_dir_user_12_group_membership_308&searchable%5B%5D=custom_fields._snipeit_max_resolutie_309&searchable%5B%5D=custom_fields._snipeit_poorten_310&searchable%5B%5D=custom_fields._snipeit_scherm_grootte_311

**Comment by snipe** [comment]
I'm more interested in app logs, to see if you're triggering a 500 error or what. Did you try clearing your browser cookies?

**Comment by cliff-LINKS** [comment]
The browser cache and cookies where cleaned.

I get the same error in Safari, Firefox and Chrome. It ends with the line '414 (Request-URI Too Long)'

**Comment by snipe** [comment]
I'm guessing you're seeing this because you have a metric butt-ton of custom fields. I'll be honest and I'm not sure how to address this right now, but we'll take a look. Some of those url parameters are needed by the table library we use, but we may be able to filter them out so they're not being sent to the server.

**Comment by cliff-LINKS** [comment]
I do have an extensive set of custom fields indeed, it did work great in version 4 though.
Thanks for looking in to this!

**Comment by snipe** [comment]
@uberbrady you're better at JS than I am - can you take a look? (I'll also be digging around). It seems bootstrap-tables is sending all of our `data-column` info to the server, but I don't see any reason why it should need to do that. The `data-columns` stuff *should* just dictate the front-end of display - which columns are searchable, sortable, etc - but *maybe* there's some additional magic that's happening there?

I'm hoping this isn't something we need to fix on the bootstrap tables end, but their documentation is.... not amazing.

**Comment by uberbrady** [comment]
@cliff-LINKS I don't know if this is related, but in that `laravel.log` file that you attached, I'm seeing a ton of errors like:

```
file_put_contents(/usr/local/snipeit/storage/framework/views/4bf8d70590170ce411f97c484fdb839346ca1a7c.php): failed to open stream: Permission denied 
```

I know the blade-caching mechanism that Laravel uses wants to be able to write into that directory. It doesn't seem very likely that that's the actual problem here, but I definitely think you should fix that.

I'm still looking into the 'having tons of custom fields' problem in case it's not related.

**Comment by snipe** [comment]
Yeah, while those permissions issues *definitely* should be fixed, the `414 (Request-URI Too Long)` is a bigger deal IMHO. BS Tables shouldn't be sending all of those extra values to the server.

**Comment by snipe** [comment]
@uberbrady if you click on the link in the example they gave, you'll see the same result. It's farting out before it can even check for auth, etc.

**Comment by uberbrady** [comment]
Yeah, even on my local setup with a small handful of custom fields, it's still sending a bunch of `searchable[]` stuff, which I don't think we want - the back-end doesn't need to know, that's just a 'hint' for the front-end, right?

**Comment by snipe** [comment]
@uberbrady yeah - the backend API is what determines whether something is actually searchable. The `searchable` data-dash attribute *should* only be used by the frontend. I could see maybe if it's using that info to determine what search string to send through? Either way, it *should* be using that to silently build the query string, not passing ALL of that shit to the server. We can experiment with removing the `searchable` attribute from the AssetsPresenter and testing to see that searching, etc still works on those fields, which would make the payload less big, but `sortable` is important to the front-end to determine whether to show the sorting arrows or not.

**Comment by snipe** [comment]
Another thought.... BS tables has the option to set `data-side-pagination="server"` or `data-side-pagination="client"`. 

I would have thought that once you set `data-side-pagination="server"` and have a `data-url=<url>`, it shouldn't try to push any of that info to the URL tho.

**Comment by conan522** [comment]
I have a similar issue, but Chorme is ok,, IE11 is empty.

**Comment by cliff-LINKS** [comment]
@uberbrady thanks for pointing me to the permission issues, they're been taken care of.

**Comment by snipe** [comment]
Can you pull the latest from `develop` and see if that helps? We've implemented a possible fix for your super-duper-long-URL issue.

**Comment by snipe** [comment]
We now have this tagged in v5.0.5 - please let me know if upgrading resolves your issue.

**Comment by cliff-LINKS** [comment]
The issue seems to be resolved in the latest build.
The version number has incremented to 5.0.6 by the time I got to testing this morning.

FYI, I had to reactivate the visible fields in the Assets view list. All be it, a non issue.

Thank you very much for looking in to this!

**Comment by snipe** [comment]
Yeah, I released v5.0.6 just a little bit ago - we added a few small features that we thought would be helpful, so we decided not to wait to tag a new release.

Glad everything is sorted now.
